### PR TITLE
[UX] ProTip to help marketers become power users

### PR DESCRIPTION
**Pull request by andersonjeccel**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 

## Description

This PR adds a template for ProTips that aim to provide extra information for users to get the most out of Mautic.

image

It's reusable and by default it accepts only a translation string for the tip, while icon and styles keep default for all.

It includes a total of 35 tips for listing pages.

---

### Tips

**Contacts**

- Use the <kbd>V</kbd> key to switch between card and table views 
- Add tags during import to help organize and segment your contacts. Example: (import_facebook_may2024) 
- Build your email list organically through opt-ins rather than purchasing lists 

**Forms**

-  Redirect URLs for forms can contain variables by using tokens like /thank-you?email={formfield=email} 
-  Minimize the number of fields to improve form submission rates. 
-  Use clear, action-oriented language in form headlines and CTAs (e.g. "Get your free guide" vs. "Submit"). 
- For longer forms, use a multi-step process to make them feel less overwhelming. 

**Tags**

- Tags can be used for quick segmentation, while custom fields are more suitable for long-term data storage. 
- Use tags instead of custom fields for multi-value contact attributes 
- You can create new tags while editing a contact. 

**Segments**

- Use behavioral data like website activity and email engagement to create dynamic segments that update automatically 
- Test different segment combinations to find the most effective groupings for your campaigns

**Assets**

- Gate valuable assets behind a form to capture lead information before allowing the download. 
- Use consistent naming conventions to keep assets organized
- Repurpose high-performing assets across different channels and campaigns
- Track asset performance metrics in the dashboard widgets to identify top-performing content

**Points**

- Implement a lead scoring decay model to accurately capture intent. 
- Reduce scores systematically over time if contacts become inactive. 
- Use a limit to prevent scores from getting too high or low 

**Triggers**

- Use behavioral triggers to send timely, relevant emails based on user actions. 

**Emails**

- Always include an unsubscribe link and never use a no-reply email address. 
- Preview and test your emails across devices and clients to ensure they look great everywhere. 
- Personalize email content as much as possible based on user behaviors and preferences 
- Avoid spam trigger words in subject lines and email content 
- Limit emails to one primary CTA to avoid decision paralysis 

**Pages**

- Optimize your website for mobile-first since most web traffic now comes from mobile devices 
- Place forms prominently "above the fold" so users don't have to scroll to find them. 

**Text messages**

- Keep messages concise and action-oriented to respect the personal nature of SMS 
- Use SMS for time-sensitive information like appointment reminders or limited-time offers 
- Comply with regulations by including opt-out instructions 

**Campaigns**

- Implement re-engagement campaigns for leads that become inactive 
- Deploy targeted survey campaigns to gather actionable insights and improve customer satisfaction 
- Use personalized cross-sell campaigns based on purchase history to increase customer lifetime value 
- Create automated onboarding sequences to guide new users through key product features and increase activation rates 

**Reports**

- Track DNC comments to reinsert contacts bounced by temporary issues. 

---
### 📋 Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open the mentioned listing pages 
3. Scroll down to the bottom
4. See the tip
5. Reload the page to see another tip

### CODE REVIEW

**Review by abhisekmazumdar — APPROVED** [review]
That good for user. Looks good to me.

**Review by LordRembo — APPROVED** [review]
The code looks good. Would be good to see what changed, after rebase + merging the open dependent PR's.

File: app/bundles/CoreBundle/Resources/views/Helper/protip.html.twig [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{% if tip is defined and tip is not empty %}
+<div class="col-xs-12 ai-center mt-md mb-md">
+    <div class="text-muted text-center">
+        <i class="ri-lightbulb-line ri-lg"></i>
+        <span class="fw-b">{{'mautic.core.protip'|trans}}</span>
```
Comment (shinde-rahul) [inline app/bundles/CoreBundle/Resources/views/Helper/protip.html.twig:5]:
```suggestion
        <span class="fw-b">{{ 'mautic.core.protip'|trans }}</span>
```

File: app/bundles/CoreBundle/Resources/views/Helper/protip.html.twig [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+        {{tip|trans|raw}}
```
Comment (shinde-rahul) [inline app/bundles/CoreBundle/Resources/views/Helper/protip.html.twig:6]:
```suggestion
        {{ tip|trans|raw }}
```

**Review by shinde-rahul — CHANGES_REQUESTED** [review]
The code looks good overall, but there are just a couple of CS issues that need to be addressed. Great improvement!

**Comment by andersonjeccel** [comment]
@shinde-rahul Thanks for reviewing!

I requested to redo after commiting the changes

**Review by shinde-rahul — APPROVED** [review]
Code LGTM!!

Thanks @andersonjeccel!!

**Review by Aleks0129 — CHANGES_REQUESTED** [review]
1. Contacts after clicking "V" pro tip isn't visible in tile view.

Screenshot 2024-09-28 at 18 52 10

2.Forms looks good.

image

3.Tags good as well.

image

4.Segments look good.

image

5.Assets good.

Screenshot 2024-09-28 at 18 57 48

6.Points good.

image

7.Triggers good.

image

8.Emails good.

image

9.Pages good.

image

10.Campaigns good.

image

11.Reports good.

image

**Comment by andersonjeccel** [comment]
@abhisekmazumdar These e2e tests seem to be really unreliable and break easily... Would you like to fix the test?

**Comment by RCheesley** [comment]
@andersonjeccel could you resolve the conflict please? Thanks!

**Comment by andersonjeccel** [comment]
Solved!

**Comment by abhisekmazumdar** [comment]
Working on fixing the test cases.

**Comment by abhisekmazumdar** [comment]
Looks like all the tests are green now. Please let me know if it fails in the future.

**Comment by andersonjeccel** [comment]
@abhisekmazumdar Thanks! 😄

**Pull request merged**